pages/index.html

```diff
@@ -74,7 +74,7 @@ <h2 class="usa-font-lead">Open source UI components and a visual style guide for
   <div class="usa-grid">
     <div class="usa-intro">
       <h2>How to use these tools</h2>
-      <h3 class="usa-font-lead">Use a little, use a lot, use it all.<br>Here’s how the web design standards may help you...</h3>
+      <h3 class="usa-font-lead">Use a little, use a lot, use it all.<br> Here’s how the web design standards may help you …</h3>
```

Comment:
I believe there is supposed to be a space before and after an ellipses. @angel—do you want to weigh in?